main.py

```diff
@@ -0,0 +1,77 @@
+from typing import List
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=schemas.AuthorList)
+def create_author(
+        author: schemas.AuthorCreate,
+        db: Session = Depends(get_db)
+):
+    db_author = crud.get_author_by_name(db=db, name=author.name)
+    if db_author:
+        raise HTTPException(status_code=400, detail="Name already registered")
+    return crud.create_author(db=db, author=author)
+
+
+@app.get("/authors/", response_model=List[schemas.AuthorList])
+def read_authors(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
+    authors = crud.get_authors(db=db, skip=skip, limit=limit)
+    return authors
+
+
+@app.get("/authors/{author_id}", response_model=schemas.AuthorList)
```

Comment:
all paths should be with `/` in the end
```suggestion
@app.get("/authors/{author_id}/", response_model=schemas.AuthorList)
```